docs/source/cheat_sheet_py3.rst

```diff
@@ -0,0 +1,280 @@
+.. _cheat-sheet-py3:
+
+Mypy syntax cheat sheet (Python 3)
+==================================
+
+This document is a quick cheat sheet showing how the `PEP 484 <https://www.python.org/dev/peps/pep-0484/>`_ type
+language represents various common types in Python 3. Up to the last section, the syntax is valid on all versions of Python 3. See the last section for more on variable annotation syntax.
```

Comment:
I'd say "Unless otherwise noted, the syntax is valid on all versions of Python 3."